migrations/0000_create_requests_table.sql

```diff
@@ -0,0 +1,19 @@
+-- Migration number: 0000 	 2023-12-20T02:07:03.645Z
+
+DROP TABLE IF EXISTS requests;
```

Comment:
Is this run every time we deploy? I don't want to lose data on deployments

Reply:
no, it will only execute once.